portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,72 @@
     <script src="script.js"></script>
   </head>
   <body>
+      <div id="header">
+      <h1>Emily Zhao</h1>
+      <h2>Hi! I'm Emily and this is my portfolio. Get ready to learn more about me!</h2>
+      </div>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h2>About Me</h2>
+      <p style="text-align:justify">I am a computer science student at Queen's University going into third year! My degree plans are to pursue a Major in Computing with a concentration in biomedical computation.</p>
+      <img src="/images/me.jpg" width="375" height="560" class="center">
+    <p style="text-align:justify">I grew up in Ottawa, Ontario with my sister and parents. I love spending time with my friends, whether it's going to music festivals, restaurants, playing wii games or even just picnics in the park.
+        I really enjoy staying active. In high school, I played soccer and basketball competitively as well as volleyball with my school team. As much as I love jam-packing my days, I also enjoy staying in
+        and catching up with what I'm currently watching on Netflix. </p>
+        <br>
+        <figure style="display:flex">
+            <img src="/images/family.jpg" width="275" height="300" style="margin: 0px 20px">
+            <img src="/images/toronto.jpg" width="275" height="300" style="margin: 0px 20px"> 
+            <img src="/images/hoco.jpg" width="275" height="300" style="margin:0px 20px">
+        </figure>
+        <br>
+        <p style="text-align:center">Click here to generate a random fact about me:</p>
```

Comment:
In the future, the JS change and this html change could be a separate PR. However, the overall size of this PR is still manageable.

You can choose to try splitting it to learn how that would work, or you can proceed with merging as-is